bulletin/index.html

```diff
@@ -203,6 +203,26 @@ <h2 class="featured-article__title"> Engineering Robust Learning Environments Th
 	  </div>
 	</section>
 
+	<section>
+		<div class="featured-article__wrapper comments-acknowledge__wrapper">
+			<div class="comments-acknowledge__section">
+				<div class="comments-acknowledge__section--heading"> 
+					<h2>Comments &</h2>
+					<h2>Acknowledgments</h2>
+				</div >
+				<div>
+					<p>This section is a tribute to our appreciation for the comments about program. It is also and acknowledgment</p>
```

Comment:
There are typos in this section, but also I think it could be more interesting. It is stating what the title/header states instead of explaining what the content/section is about. 

For example, "We want to acknowledge the support from our generous donors that make the program possible" 

Something like that or else I would say remove it.

Reply:
You can make the modifications that you feel will make the reading more interesting. I am fine with that. I am not a creative writer.